### ISSUE TITLE
RGTC BC4 Signed RED1 calculation might be wrong

### ISSUE THREAD

**Issue by Qining** [KhronosGroup/DataFormat-Registry#1]
In the pdf and html, it says:
```
RED1 = 
  red1/127.0, red1 > -128
  -1.0, **red0** = -128
```
Probably it should be
```
RED1 = 
  red1/127.0, red1 > -128
  -1.0, **red1** = -128
```

Doubled checked with https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_texture_compression_rgtc.txt
And it seems it should be **red1** instead of **red0**

**Comment by fluppeteer** [comment]
Oops. Thank you. A large slew of typographical changes are in flight; I'll fix this (and the other outstanding typos) as soon as I can incorporate it.

**Comment by MarkCallow** [comment]
This is fixed in the just published KDFS 1.3. @oddhack please close this issue.
